### PR TITLE
Draft features

### PR DESCRIPTION
**Pull request by Gnwin**
Changes from the last review have been effected. awaiting approval.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+                                <div class="vs2">
+                                    <div class="title">Samuels Football</div> 
+                                    <div class="subscribe">
+                                        <button><i class="fa fa-youtube-play fa-lg" aria-hidden="true"></i>Subscribe</button>
+                                        <div class="num">799</div>
+                                    </div>
+                                </div>
+                            </div>
+                            
+                            <div class="views" style="display: flex; align-items: flex-end;">
```
Comment (HADDADSOHAIB) [inline index.html]: Inline style here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+                    </div>
+                    <div class="load" style="text-align: center; color:#7D7D7D; cursor: pointer;"><i class="fa fa-spinner fa-2x" aria-hidden="true"></i> Loading</div>
+                </div>
+                
+                <aside class="more-vid">
+                    <div class="more">
+                        <a href="" class="upnext">Up next</a>
+                        <div class="" style="float:right">
+                            <a href="" class="auto" style="color: #ABABAB; font-weight: bolder;">Autoplay</a>
+                            <i class="fa fa-exclamation-circle" style="color: #808080;"></i>
```
Comment (HADDADSOHAIB) [inline index.html]: inline styles here, also remove empty classes (class="")

File: index.html [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+                            <i class="fa fa-toggle-on fa-lg"></i>
+                        </div>
+                    </div>
+                    <div class="suggest-vid">
+                        <div class="vid">
+                            <a href=""><img src="http://i3.ytimg.com/vi/adseCgsYSYs/maxresdefault.jpg" style="width: 100%;" alt=""></a>
```
Comment (HADDADSOHAIB) [inline index.html]: Inline styles here

File: index.html [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+                        <div class="vid-info">
+                            <p class="title">Best of LeBron James So Far This Season</p>
+                            <p class="channel">NBA</p> 
+                            <p class="views">29,985 views</p>
+                        </div>
+                    </div>
+
+                    <div class="suggest-vid">
+                        <div class="vid">
+                            <a href=""><img src="http://i3.ytimg.com/vi/adseCgsYSYs/maxresdefault.jpg" style="width: 100%;" alt=""></a>
```
Comment (HADDADSOHAIB) [inline index.html]: Inline style here

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hello @Gnwin and @rashidmvk    :hand: 

### Good job, you did great job with the project :+1: 

### Unfortunately, there are still some changes to get the project approved:
- This is a rereview, and I don't see the previous review here, please provide a link to the previous pull request. You should not change the pull request unless it is marked as invalid.
- In your pull request, provide descriptive comments (We implemented in this project a clone for youtube video player .....).
- This marked as a two-person project, but I see one person is committing, please explain why is that happening.
- Please remove inline styles, check my comments for details, all you styles would be in css files.
- You need to make this part like the original project:
Your project:
Screenshot from 2020-06-16 01-03-11
Original:
Screenshot from 2020-06-16 01-04-11
### Once you are done with the changes, please ask  for another code review.

File: index.html (hunk 2 above)
Comment (Gnwin) [reply index.html]: all done

**Pull request closed without merging**